test_fractions.py

```diff
@@ -796,11 +797,50 @@ def test_pi_digits(self):
             self.assertEqual(ff.numerator, qf.numerator)
             self.assertEqual(ff.denominator, qf.denominator)
 
+class CImportTest(unittest.TestCase):
+
+    def setUp(self):
+        self.build_test_module()
+
+    def tearDown(self):
+        self.remove_test_module()
+
+    def build_test_module(self):
+        self.module_code = '\n'.join([
+            '# cython: language_level=3str',
+            'from quicktions cimport Fraction',
+            'def get_fraction():',
+            '    return Fraction(1, 2)',
+        ])
+        self.base_path = os.path.abspath(os.path.dirname(__file__))
+        self.module_name = 'quicktions_importtest'
+        self.module_filename = os.path.join(self.base_path, '.'.join([self.module_name, 'pyx']))
+        with open(self.module_filename, 'w') as f:
+            f.write(self.module_code)
+
+    def remove_test_module(self):
+        for fn in os.listdir(self.base_path):
+            if not fn.startswith(self.module_name):
+                continue
+            os.remove(os.path.join(self.base_path, fn))
+
+    def test_cimport(self):
+        self.build_test_module()
+        import pyximport
+        self.py_importer, self.pyx_importer = pyximport.install(inplace=True, language_level=3)
```

Comment:
Please don't use pyximport for the tests. It's really outdated.
[cython.inline()](http://docs.cython.org/en/latest/src/userguide/source_files_and_compilation.html#compiling-with-cython-inline) might work for this.

Reply:
I did look into that as it would be much cleaner, but from what I've tested, `cimport` isn't supported:

```python
import cython
r = cython.inline('from quicktions cimport Fraction;return Fraction(a,b)', a=1,b=2)
```
Raises an exception:

```
AssertionError: Not yet supporting any cimports/includes from string code snippets
```

I agree that there should be a better way though